src/internal/converters/eml/eml.go

```diff
@@ -0,0 +1,142 @@
+package eml
+
+// This package helps convert from the json response
+// received from Graph API to .eml format (rfc0822).
+// Ref: https://www.ietf.org/rfc/rfc0822.txt
+// Ref: https://datatracker.ietf.org/doc/html/rfc5322
+// Data missing from backup:
+// SetReturnPath SetPriority SetListUnsubscribe SetDkim
+// AddAlternative SetDSN (and any other X-MS specific headers)
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/alcionai/clues"
+	"github.com/microsoftgraph/msgraph-sdk-go/models"
+	mail "github.com/xhit/go-simple-mail/v2"
+
+	"github.com/alcionai/corso/src/internal/common/ptr"
+	"github.com/alcionai/corso/src/pkg/logger"
+)
+
+const (
+	fromFormat = "%s <%s>"
```

Comment:
nit: maybe a more generic name would work better since it's used for senders and receivers? Something like `nameAddressFormat` maybe? (idk, naming is hard)

Reply:
edit: still making my way through PRs, but seems like this may get renamed later?

Reply:
Should be fixed in https://github.com/alcionai/corso/pull/4651